src/ESP_FlexyStepper.cpp

```diff
@@ -91,21 +91,21 @@ ESP_FlexyStepper::~ESP_FlexyStepper()
 }
 
 //TODO: use https://github.com/nrwiersma/ESP8266Scheduler/blob/master/examples/simple/simple.ino for ESP8266
-void ESP_FlexyStepper::startAsService(bool core)
+void ESP_FlexyStepper::startAsService(int core)
```

Comment:
we still need this check, otherwise invalid values will be silently ignored and this can lead to problems in troubleshooting.
If the user provides invalid values he need to be warned about it somehow.
Your approach would rely on the implementation of the undocumented function xTaskCreateUniversal which silently ignores all other values and will just create the tasks on the default core without letting the developer know that he provided an invalid vlaue.

Reply:
Alternatively, we can simply copy the contents of xTaskCreateUniversal to create our task, if you are afraid of some problems, although this function is used in the library https://github.com/me-no-dev

Reply:
No, it's fine, we give it at try and use the existing function. Copying would be worse :-)